src/irr/asset/CImageWriterOpenEXR.cpp

```diff
@@ -77,44 +77,30 @@ namespace irr
 				if (region->imageSubresource.mipLevel == 0)
 				{
 					regionsToHandle.push_back(region);
-					width += region->bufferRowLength;
-					height += region->bufferImageHeight;
+					width += region->imageExtent.width;
+					height += region->imageExtent.height;
```

Comment:
nope, you can get ths from image creation parameters

Reply:
but won't it affect then with another mipmap levels? 
unless image creation parameters extent holds the buffer assuming 0 mipmap level for each region

Reply:
image creation parameters must create an image big enough so all regions in the list are valid (i.e. can fit inside)